galata/test/jupyterlab/launcher.test.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, galata, test } from '@jupyterlab/galata';
+
+test.use({
+  autoGoto: false,
+  mockState: galata.DEFAULT_DOCUMENTATION_STATE,
+  viewport: { height: 720, width: 1280 }
+});
+
+test.describe('Dynamic Text Spacing', () => {
+  test('should Use Dynamic Text Spacing', async ({ page }) => {
+    await page.goto();
+    await page.waitForSelector('.jp-LauncherCard-label');
+
+    let element = page.locator('div.jp-LauncherCard-label');
+    for (let i = 0; i < (await element.count()); i++) {
+      let height = await element
+        .nth(i)
+        .evaluate(el =>
+          window.getComputedStyle(el).getPropertyValue('min-height')
+        );
+
+      let expectedValue =
+        2.462 *
+        parseFloat(
+          await element
+            .nth(i)
+            .evaluate(el =>
+              window
+                .getComputedStyle(el)
+                .getPropertyValue('font-size')
+                .toString()
+            )
+        );
+
+      expect(height).toEqual(expectedValue + 'px');
+    }
+
+    const imageName = 'launcher-card-label-height.png';
+    expect(await page.screenshot()).toMatchSnapshot(imageName.toLowerCase());
```

Comment:
Can you please change it so that only the launcher cards are in the screenshot (please select `.jp-Launcher-content`)? Otherwise this test will fail for example if there is a change to the file browser which and then on the update the gist of it may be lost.

Reply:
Hi @krassowski I have changes the screenshot as you have suggested. Please have a look. Thank you!

Reply:
Hi @krassowski, i have made the changes you have asked for. Please have a look.